src/components/aktivitetskrav/vurdering/VurderAktivitetskravModal.tsx

```diff
@@ -62,37 +64,21 @@ export const ModalContent = styled.div`
 `;
 
 const VurderAktivitetskravModalContent = ({
-  setModalOpen,
   modalType,
-  aktivitetskravUuid,
-}: VurderAktivitetskravModalContentProps) => {
+  ...rest
```

Comment:
Hva menes med rest?

Reply:
Det er vel "resten" av propsene, som å skrive `...props`, men så tror jeg man kan kalle det hva man vil (feks `...rest`) 🤷🏼‍♂️

Reply:
Resten av propsene (i dette tilfellet `setModalOen` og `aktivitetskravUuid`) Kunne kalt det noe annet også, men tror det er sånn halvveis konvensjon når man gjør "destructuring" av props (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for å sende de videre til "children" komponenter :)

Reply:
> Det er vel "resten" av propsene, som å skrive `...props`, men så tror jeg man kan kalle det hva man vil (feks `...rest`) 🤷🏼‍♂️

Ja, bruker vel typisk `...props` hvis man vil gjøre "spread" av alle props, mens `..rest` når man plukker noen props og sender "resten" videre.